web/themes/custom/contribtracker/cypress.config.js

```diff
@@ -4,4 +4,5 @@ export default defineConfig({
   e2e: {
     baseUrl: 'https://contribtracker.ddev.site/',
   },
+  video: true,
```

Comment:
Do we have to explicitly say this? What about screenshots? I ask because I recall not seeing screenshots in some of the recent artifacts.

Reply:
It only captures screenshots if a test fails. From version 13, capturing video is turned off by default, so I had to enable it.

I am trying to find out what changed in version 13 that stopped screenshots from being taken for each test. I am just curious how Cypress decides when to take a screenshot for a successful test, as a test may contain multiple actions expanding to 4 to 5 seconds or more.

Reply:
No, I don't see any attributes for capturing screenshots. We can capture screenshots manually in the code using `[cy.screenshot](https://docs.cypress.io/api/commands/screenshot)` if required. 
Anyway, it captures videos and screenshots in case of failure, so it serves our purpose for now.